config/weekly_checks.yaml

```diff
@@ -0,0 +1,36 @@
+version: 0.0.1
+name: 'Weekly Checks'
+
+input:
+  files:
+    - data/L1Ntuple_*.root
+  sample:
+    name: Data
+    title: 2016 Data
+  trigger:
+    name: SingleMu
+    title: Single Muon
+  pileup_file: ""
+  run_number: 276243
+
+analysis:
+  do_fit: False
+  pu_type: 0PU12,13PU19,20PU
+  pu_bins: 0,13,20,999
+  analyzers:
+    - cmsl1t.analyzers.energy_sums_efficiencies
+    - cmsl1t.analyzers.jet_efficiencies
```

Comment:
I've spoken in person to Alex now and said that these aren't the "contemporary" form of analyzer, and that he should base his around the `study_met` one.

Reply:
OK, let's change it up. I am thinking of creating a new branch `weekly_config` in here so we can do quick iterations.

Reply:
Good shout.  For now, based on the list of plots on issue #65, I've suggested we just have a single analyzer that runs things.  We'll have to keep an eye on how that evolves though, and potentially generalize that analyze as needs develop.